test/account/extensions/ERC7579Modules/SocialRecoveryExecutorModule.test.js

```diff
@@ -0,0 +1,708 @@
+const { expect } = require('chai');
+const { ethers, entrypoint } = require('hardhat');
+const { loadFixture, time } = require('@nomicfoundation/hardhat-network-helpers');
+const { impersonate } = require('@openzeppelin/contracts/test/helpers/account');
+const {
+  MODULE_TYPE_EXECUTOR,
+  encodeMode,
+  CALL_TYPE_SINGLE,
+  EXEC_TYPE_DEFAULT,
+  encodeSingle,
+} = require('@openzeppelin/contracts/test/helpers/erc7579');
+const { PackedUserOperation } = require('../../../helpers/eip712-types');
+const { ERC4337Helper } = require('../../../helpers/erc4337');
+const { SocialRecoveryExecutorHelper } = require('../../../helpers/erc7579-modules');
+const { SIG_VALIDATION_SUCCESS, SIG_VALIDATION_FAILURE } = require('@openzeppelin/contracts/test/helpers/erc4337');
+const { getDomain } = require('@openzeppelin/contracts/test/helpers/eip712');
+
+async function fixture() {
+  // ERC-7579 validator
+  const validatorMock = await ethers.deployContract('$ERC7579ReconfigurableValidatorMock');
+
+  // ERC-4337 signers
+  const initialSigner = ethers.Wallet.createRandom();
+  const newSigner = ethers.Wallet.createRandom();
+
+  // ERC-4337 account
+  const helper = new ERC4337Helper();
+  const env = await helper.wait();
+  const accountMock = await helper.newAccount('$AccountERC7579Mock', [
+    'AccountERC7579',
+    '1',
+    validatorMock.target,
+    ethers.solidityPacked(['address'], [initialSigner.address]),
```

Comment:
That should be just the same
```suggestion
    initialSigner.address,
```